sculture-android/app/src/main/java/tr/edu/boun/cmpe/sculture/activity/ProfilePageActivity.java

```diff
@@ -0,0 +1,161 @@
+package tr.edu.boun.cmpe.sculture.activity;
+
+/**
+ * Created by SahaOperasyon2 on 06.12.2015.
+ */
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import tr.edu.boun.cmpe.sculture.R;
+import tr.edu.boun.cmpe.sculture.adapter.StoryListViewAdapter;
+import tr.edu.boun.cmpe.sculture.models.response.BaseStoryResponse;
+import tr.edu.boun.cmpe.sculture.models.response.SearchResponse;
+
+import static tr.edu.boun.cmpe.sculture.BaseApplication.baseApplication;
+import static tr.edu.boun.cmpe.sculture.Constants.API_USER_STORIES;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_ID;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_PAGE;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_SIZE;
+import static tr.edu.boun.cmpe.sculture.Constants.PREF_USER_ID;
+import static tr.edu.boun.cmpe.sculture.Utils.addRequest;
+
+public class ProfilePageActivity extends AppCompatActivity {
```

Comment:
The activity should request an user id by its bundle.
Also the activity should load the profile of this user.
There is no request about the user on this activity. The views are not filled.